MetaScriptToolbox/script.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "MetaScriptToolbox",
+  "script": "MetaScriptToolbox.js",
+  "version": "1.0.0",
```

Comment:
After looking through these changes, should this be version `0.0.1`?

Reply:
Good catch... updated files to match the 0.0.1 version throughout. Files should be in the branch now.